app/decorators/user_decorator.rb

```diff
@@ -76,7 +76,6 @@ def config_body_class
     body_class = [
       setting.config_theme.tr("_", "-"),
       "#{setting.resolved_font_name.tr('_', '-')}-article-body",
-      "trusted-status-#{trusted?}",
```

Comment:
This one should remain in the decorator as per the suggestion mentioned above

Reply:
Reverted this change